012-sorting

seemaPatl opened the pull request:
Hello @skully-coder , I have added 012 array sorting under string folder, please accept the merge with the tag "hacktoberfest-accepted
".

skully-coder reviewed (changes requested):
Hello @seemaPatl !
Please provide a README as well

seemaPatl commented:
> Hello @seemaPatl ! Please provide a README as well

Hi @skully-coder , I have added the required files, please add the merge. Thank you. 😊

skully-coder reviewed (changes requested):
@seemaPatl 
Please add a readme for String/012-pattern as well

skully-coder commented:
@seemaPatl
Please add a readme for String/012-pattern as well, or else I will have to close this PR

seemaPatl commented:
> @seemaPatl Please add a readme for String/012-pattern as well, or else I will have to close this PR

Hi @skully-coder , I have added the required files, please add the merge. Thank you. 😊. Last night only I have added the read.me, please check.
